### ISSUE TITLE
social worker notes-taking

### ISSUE THREAD

**Issue by johncoffee** [SOSChildrensVillages/HackXmas2019#10]
Hi! 
Our group explores the problems related to collecting notes from fieldwork by the social workers.

Could you say that there are certain formulas they fill out regularly that are the same?
eg. Are there a basic set of questions they always collect from the families?

What would it look like when the social worker go out and take notes on the families progress over time? 
eg. Are those data points consistent?

**Comment by strabu** [comment]
Yes, they do the "Core Assessment" as part of their Case Management approach - see: documentation/screenshots-pdb2/2b_PDB-screenshot-CoreAssessment.png which is based on Indicators that spread across 8 dimensions.
In each Area they can type text/comments at the bottom but most is captured with a 1-4 scale that is tracked over time.

**Comment by strabu** [comment]
The "Core Assessment" must be done once a year. (this is monitored with reports) -> The App should remind responsible social workers when an Assessment is due.
